<ui version="4.0" >
 <class>tke_Form_fastPilgovykyDocInsert</class>
 <widget class="QWidget" name="tke_Form_fastPilgovykyDocInsert" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Швидке внесення документів на пільги</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <property name="spacing" >
    <number>2</number>
   </property>
   <property name="margin" >
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_pilgKategorFiltr" >
       <property name="text" >
        <string>Фільтр по катнгорії пільги</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_pilgKategorFiltr" >
       <property name="minimumSize" >
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_pilgovykyDocs" >
     <property name="editTriggers" >
      <set>QAbstractItemView::AllEditTriggers</set>
     </property>
     <column>
      <property name="text" >
       <string>id</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Здано</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Рахунок</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Прізвище</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>І. та П.</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Адреса</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Кат.пільги</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_pilgovykyCount" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
